src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,57 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int MAX_SIZE = 12;
+    private static final int EMPTY = -1;
+    private static final int FULL = 11;
```

Comment:
This constant is related to MAX_SIZE. Changing only one leads to an error (inconsistency).